tests/UI/campaigns/functional/BO/05_customerService/03_merchandiseReturns/01_activateMerchandiseReturn.js

```diff
@@ -0,0 +1,177 @@
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  const tests = [
+    {args: {action: 'activate', enable: true}},
+    {args: {action: 'deactivate', enable: false}},
+  ];
+
+  tests.forEach((test, index) => {
+    it('should go to merchandise returns page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `goToMerchandiseReturnsPage${index}`, baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.customerServiceParentLink,
+        dashboardPage.merchandiseReturnsLink,
+      );
+
+      await merchandiseReturnsPage.closeSfToolBar(page);
+
+      const pageTitle = await merchandiseReturnsPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(merchandiseReturnsPage.pageTitle);
+    });
+
+    it(`should ${test.args.action} merchandise returns`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}Returns`, baseContext);
+
+      const result = await merchandiseReturnsPage.setOrderReturnStatus(page, test.args.enable);
+      await expect(result).to.contains(merchandiseReturnsPage.successfulUpdateMessage);
+    });
+
+    it('should go to orders page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `goToOrdersPage${index}`, baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.ordersParentLink,
+        dashboardPage.ordersLink,
+      );
+
+      const pageTitle = await ordersPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(ordersPage.pageTitle);
+    });
+
+    it('should filter the Orders table by the default customer \'J. DOE\' and check the result', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `filterOrder${index}`, baseContext);
+
+      await ordersPage.filterOrders(page, 'input', 'customer', 'J. DOE');
+
+      const textColumn = await ordersPage.getTextColumn(page, 'customer', 1);
+      await expect(textColumn).to.contains('J. DOE');
+    });
+
+    it('should go to the first order page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `goToOrderPage${index}`, baseContext);
+
+      // View order
+      await ordersPage.goToOrder(page, 1);
+
+      const pageTitle = await viewOrderPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(viewOrderPage.pageTitle);
+    });
+
+    it('should change the order status to \'Shipped\' and check it', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `updateOrderStatus${index}`, baseContext);
+
+      const result = await viewOrderPage.modifyOrderStatus(page, 'Shipped');
+      await expect(result).to.equal('Shipped');
```

Comment:
Shouldn't that be handled at the page class level ?

Reply:
Done thx :+1: